src/UI/components/map/map.tsx

```diff
@@ -64,25 +66,27 @@ export const MapWrapper = () => {
   const mapElement = useRef(null);
 
   useEffect(() => {
-    const markStyle = new Style({
-      image: new Icon({
-        scale: 0.4,
-        crossOrigin: 'anonymous',
-        src: 'icons/marker.png',
-      }),
-      text: new Text({
-        text: 'Test text',
-        scale: 1.2,
-        fill: new Fill({
-          color: '#fff',
+    function markStyle(n_all: number, species: string) {
+      return new Style({
+        image: new Circle({
+          radius: 15,
+          fill: new Fill({
+            color: speciesColorMapRGB(speciesObject.data, species).color,
```

Comment:
I assume this means the redux state has the `data` field? Can we deconstruct this before it's saved in the state, so we just save the species list?